kaniko.go

```diff
@@ -224,7 +227,12 @@ func (p Plugin) Exec() error {
 	if p.Build.TarPath != "" {
 		cmdArgs = append(cmdArgs, fmt.Sprintf("--tar-path=%s", p.Build.TarPath))
 	}
-	
+
+	//Read all PLUGIN_ env vars
+	//parse them such that PLUGIN_ENV_ARG is set to the value of --env-arg
+	//Add the value of --env-arg to cmdArgs if it does not exist
+	cmdArgs = getPLUGINEnvVars(cmdArgs)
```

Comment:
Can you change the name to `getPluginEnvVars`

Reply:
done